src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/README.md

```diff
@@ -0,0 +1,163 @@
+# GraphQL API
```

Comment:
- Anything about all the ways to query content (published, latest, ...). Some examples with fields maybe? 
- What about explaining how to query with SQL/Lucene Queries? Or maybe in these modules' documentation? Also what is the format of the Schema string that can be set in these?